desc/__init__.py

```diff
@@ -101,10 +102,162 @@ def set_device(kind="cpu"):
         ) / 1024  # in GB
         os.environ["CUDA_VISIBLE_DEVICES"] = str(selected_gpu["index"])
+
+
+logging.getLogger().addHandler(logging.NullHandler())
+# DESC automatically configures it's own logger, not the root logger, so we give the
+# root logger a NullHandler.  DESC's logger config can be changed with the following
+# functions, or with flags passed to any input-reading desc function as desired, with
+# the 'quiet' (-q), or 'disable_logging' (-d) flags to prevent messages propogating
+# into your own logging solution.  'disable logging' will toggle logfile logging off,
+# and the 'quiet' command will turn off console logging.
+
+
+def stop_logfile_logging():
+    """Quickly stops logfile logging to DESC logger.
+
+    Returns
+    -------
+        bool :
+            Returns True if logfile logger was successfully turned off.
+    """
+    logger = logging.getLogger("DESC_logger")
+    if logger.handlers is not None:
+        for handler in logger.handlers:
+            if isinstance(handler, logging.handlers.RotatingFileHandler):
+                handler.setLevel("CRITICAL")
+                return True
+            else:
+                warnings.warn(
+                    colored("No DESC logfile logger has been found.", "yellow")
+                )
+    else:
+        warnings.warn(colored("No DESC logger has been found.", "yellow"))
+        return False
+
+
+def set_logfile_logging(logfile_level="DEBUG", logfile_file="desc.log"):
+    """Quickly adds a logfile handler to the DESC logger.
+
+    Parameters
+    ----------
+    logfile_file : str
+        path to, and filename of, logfile to write output to
+    logfile_level : str
+        level of logging to logfile; "DEBUG", "INFO", WARNING", "ERROR" or
+        "CRITICAL" are accepted values, in increasing order of severity. DESC
+        only uses "DEBUG" and "INFO" currently.
+
+    Returns
+    -------
+    logger : Logger object
+        Returns Logger object if successfully configured
+    """
+    # Creates logger that accepts DEBUG level logs and up
+    logger = logging.getLogger("DESC_logger")
+    logger.setLevel(logfile_level)
+
+    if logger.hasHandlers:
+        for handler in logger.handlers:
+            if isinstance(handler, logging.handlers.RotatingFileHandler):
+                logger.removeHandler(handler)
+
+    # Create file handler
+    logfile_handler = logging.handlers.RotatingFileHandler(
+        logfile_file, maxBytes=5 * 1024 * 1024, backupCount=1, mode="w"
+    )
+    try:
+        logfile_handler.setLevel(logfile_level)
+    except ValueError:
+        warnings.warn(
+            colored(
+                "Failed to set log file log level: invalid level: '%s'" % logfile_level,
+                "yellow",
+            )
+            + "use 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL', or an integer"
+            + "value from 0 to 50"
+        )
+        return
+
+    # Set log formatting
+    logfile_formatter = logging.Formatter(
```

Comment:
Can you change this so that the logging output is more in line with what we have when it prints to std out?